packages/inventory-general-info/src/selectors.js

```diff
@@ -27,8 +28,9 @@ export const propertiesSelector = ({
         ...mount_point && safeParser(mount_point, 'mountpoint'),
         ...options && safeParser(options, 'options'),
         ...type && safeParser(type, 'mounttype')
-    })
-    )
+    }),
+    ),
+    sap_sids
```

Comment:
I'd rename this to `sapIds: sap_sids` so we don't have to disable cammel case all the time.